features/steps/partcad-cli/commands/healthcheck/common.py

```
@@ -1,15 +1,15 @@
-from click.testing import CliRunner
 from contextlib import ExitStack
+from click.testing import CliRunner
 
 from partcad_cli.click.command import cli
 import partcad.logging  as pc_logging
 
 runner = CliRunner(mix_stderr=False)
 
-@when("I run partcad healthcheck")
-def step_impl(context):
+def healthcheck_cli(context, options: List[str] = []):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)